.gitignore

```diff
@@ -1,6 +1,7 @@
 # npm
 node_modules
 package-lock.json
+npm-shrinkwrap.json
```

Comment:
Note that Im adding shrinkwrap to gitignore, we don't generate it, but we did go out of our way in the CI before to configure node to ignore it

Comfortable removing that config part, and just making sure we never commit it